beast/observationmodel/noisemodel/toothpick.py

```diff
@@ -375,10 +345,9 @@ def fit_bins(
 
             d = self._compute_sigma_bins(
                 mag_in,
-                magflux_out,
+                flux_out,
                 nbins=nbins,
-                completeness_mag_cut=completeness_mag_cut,
-                ast_nonrecovered_frac=ast_nonrecovered_frac,
+                ast_nonrecovered_ratio=ast_nonrecovered_ratio,
```

Comment:
should this be included as a parameter in the beast_settings? Where can the users set this ratio?

Reply:
Not sure.  It is a bit of an ad hock parameter and I'm wondering if we should let it be accessible to the general user.  It is possible to change if if you are an expert user via the code.  So, I'm inclined to leave it as it is right now.

Reply:
I actually agree with you. Then another question pops out. Why do we want to consider a case where ast_nonrecovered_ratio is None? Not a big deal though. :)

Reply:
That case is already included - check out line 188 in the same file.